_drafts/oryx/setup.md

```diff
@@ -0,0 +1,58 @@
+# Setup
+
+This guide explains how to set up your environment for Oryx framework development.
+
+## Prerequisites
+
+To use the Oryx framework, you should be familiar with the following:
+
+- Javascript
+- CSS
+- HTML
+
+Knowledge of Typescript will be a plus but it's not required.
+
+To install Oryx on your local system, you need the following:
+
+- active LTS or maintenance LTS version of [Node.js](https://nodejs.org/)
+- npm package manager
+- [Git](https://git-scm.com/) version control system
+
+Also you need to have [Spryker backend](https://docs.spryker.com/docs/scos/dev/setup/setup.html) up and running.
```

Comment:
you need it locally?

---

theoretically, for storefront, you only need **GLUE API** up and running **somewhere**, other stuff is not so important, right?

Reply:
Yes, I'd expect a section on the env file that we have:
- explain the variables, explain that you can connect your custom backend
- explain the standard file 
honestly, it would be even better to make a link to a seperate page that describes the env.